crates/bevy_animation/src/util.rs

```diff
@@ -0,0 +1,10 @@
+/// Steps between two different discrete values of any clonable type.
+/// Returns a copy of `b` if `t >= 1.0`, otherwise returns a copy of `a`.
+#[inline]
+pub(crate) fn step_unclamped<T>(a: T, b: T, t: f32) -> T {
+    if t >= 1.0 {
+        a
+    } else {
+        b
+    }
+}
```

Comment:
```suggestion
/// Steps between two different discrete values of any type.
/// Returns `a` if `t < 1.0`, otherwise returns `b`.
#[inline]
pub(crate) fn step_unclamped<T>(a: T, b: T, t: f32) -> T {
    if t < 1.0 {
        a
    } else {
        b
    }
}
```
Fixed the docs, and the code was returning the values in the wrong order.
Not sure what `unclamped` is referring to, tempted to call this `step_end` instead